supervision/detection/core.py

```diff
@@ -710,6 +710,57 @@ def from_paddledet(cls, paddledet_result) -> Detections:
             class_id=paddledet_result["bbox"][:, 0].astype(int),
         )
 
+    @classmethod
+    def from_depthai(cls, depthai_results) -> Detections:
+        """
+        Creates a Detections instance from a
+        [DepthAI](https://https://github.com/luxonis/depthai) inference result.
+
+        Args:
+            depthai_results (depthai/resources/nn):
+                The output Detections instance from DepthAI
+
+        Returns:
+            Detections: A new Detections object.
+
+        Example:
```

Comment:
Could we make this example a bit simpler? Maybe show how it works for a single frame. For sure, drop tracker.